src/js/components/project-view/empty-project.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
 
-import { Project } from '../../modules/projects';
+import Icon = require('react-fontawesome');
 
+import { Project } from '../../modules/projects';
 import SectionTitle from '../common/section-title';
-import SetupInstructions from './setup-instructions';
+import { selectText } from './helpers';
```

Comment:
I recommend rebasing onto the latest version of master. It includes usage of `selectText` in other places as well where this could be used. Since it's used outside of the Project View in those cases and since it's not a component, it might make sense to move `helpers.ts` to `src/js/`.